keyboards/pabile/p20/README.md

```diff
@@ -0,0 +1,16 @@
+# P20
```

Comment:
Could you please rename these README files to lowercase?

Reply:
^ This.

Specifically, we have users with a mix of operating systems, and some OS's are better at handling cases than others. 

To prevent issues from cropping up, all lowercase makes things easier.

Reply:
renamed. thanks!